gm4_lightning_rods/data/lightning_rods/functions/convert_creeper_spawner.mcfunction

```diff
@@ -0,0 +1 @@
+data merge block ~ ~ ~ {MaxSpawnDelay:801s,SpawnData:{id:"minecraft:creeper",powered:true},SpawnPotentials:[{Entity:{id:"minecraft:creeper",powered:true},Weight:1}]}
```

Comment:
instead of calling this function in `#lightning_rods:boom` it'd be quicker just to add this command to `#lightning_rods:boom`